src/nomos/agent/STRINGS.in

```diff
@@ -12209,6 +12209,10 @@ k
 %KEY% "(\<gnu|free\>|l?gpl)"
 %STR% "(lesser|library) (gpl|general public licen[cs]e)"
 #
+%ENTRY% _TEXT_MPLV2
+%KEY% "licen[cs]"
+%STR% "MPL.?([\/ -])?2([\.?0])?"
```

Comment:
Shouldn't the regex be like bellow to capture `MPL-2.0` text?
```suggestion
%STR% "MPL.?([\/ -])?2(\.?0)?"
```

Because right now, it is capturing only `MPL-2.` part of `MPL-2.0`

![image](https://user-images.githubusercontent.com/18077542/121841449-e2459280-ccfb-11eb-944d-782c56a69208.png)

Reply:
Keeping the character set `[]` also changes the `?` to a literal rather than optional quantifier.